src/rms_cluster_manager.erl

```diff
@@ -81,6 +83,36 @@ get_cluster_riak_config(Key) ->
 get_cluster_advanced_config(Key) ->
     rms_cluster:get_field_value(advanced_config, Key).
 
+-spec get_cluster_structure(rms_cluster:key()) ->
+    {ok, [{atom(), term()}]} | {error, term()}.
+get_cluster_structure(Key) ->
+    case get_cluster(Key) of
+        {ok, Cluster} ->
+            NullIfEmptyValue = fun(Name, PropList) ->
```

Comment:
Is there a reason we don't store an unset riak.conf or advanced.config as `undefined` in the cluster data, rather than an `<<>>`? If we did, you could avoid this fun and simply map `undefined` -> `null`.